astropy/uncertainty/core.py

```diff
@@ -220,6 +220,21 @@ def pdf_percentiles(self, percentile, **kwargs):
             return self.distribution._new_view(percs)
         else:
             return percs
+            
+    def pdf_asymstd(self):
+        """ 
+        a convenience function that computes the 50% percentile
```

Comment:
```suggestion
        A convenience function that computes the 50% percentile
```

Reply:
@eteq  thanks for the change req ,i have changed the files as 
 mentioned